CONTRIBUTING.rst

```diff
@@ -41,7 +41,7 @@ Talk to Us
 
 Have questions or need help?
 
-* If you want to follow our development plans, our main issue tracker is https://clusterhq.atlassian.net.
+* If you want to follow our development plans, our main issue tracker is https://clusterhq.atlassian.net/secure/Dashboard.jspa.
```

Comment:
I'm not sure I like this change.  `https://clusterhq.atlassian.net/` seems like a better canonical URL to refer people to.  For all I know, Atlassian will change the redirect to point to `Dashboard.jspx` tomorrow and this will become a 404.

I take it there's no easy way to make the link checker allow URLs that are 302s?